src/sfr.cpp

```diff
@@ -149,7 +149,6 @@ namespace sfr {
         SFRField<uint32_t> step_time = SFRField<uint32_t>(0, 0x2006, constants::eeprom::camera_step_time_offset, true);
         SFRField<uint32_t> init_start_time = SFRField<uint32_t>(0, 0x2007, constants::eeprom::camera_init_start_time_offset, true);
         SFRField<uint32_t> init_timeout = SFRField<uint32_t>(12000, 0x2008, constants::eeprom::camera_init_timeout_offset, true);
-        SFRField<uint32_t> begin_delay = SFRField<uint32_t>(100, 0x2009, constants::eeprom::camera_begin_delay_offset, true);
```

Comment:
Offsets need to be adjusted in constants.hpp -> (camera_begin_delay_offset). Please reach out to Eric if you have questions about what to change.

Reply:
Removed this offset, does everything look good?